### ISSUE TITLE
Windows: hitting arrow keys on year select without first opening dropdown causes odd results

### ISSUE THREAD

**Issue by WickyNilliams** [duetds/date-picker#11]
**Describe the bug**
On windows, if you do not open the year dropdown but use arrows keys, the selected year jumps around by more than one year at a time.

**To Reproduce**
1. open calendar
2. tab to year dropdown
3. hit up/down keys 
4. observe undesirable behaviour:

```
Start -> 2020
Down -> 2021 (+1)
Down -> 2023 (+2)
Down -> 2026 (+3)
Up -> 2028 (+2)
Up -> 2029 (+1)
Up -> 2029 (+0)
Up -> 2028 (-1)
Up -> 2026 (-2)
Down -> 2025 (-1)
```

**Expected behavior**
Years increment/decrement one year at a time

**Desktop (please complete the following information):**
 - OS: Windows 	
 - Browser:  Edge
 - Version: 85.0.564.51

**Comment by bodinaren** [comment]
There's something else that's weird with the year dropdown which seems related. This time I'm using the mouse instead of the keyboard.

1. Open the dropdown
2. Select another year
3. Open dropdown again
4. The year pre-selected is not the year you selected.

The pre-selected year appears to be twice of what you selected, so if you selected 2021 it pre-selects 2022. 2022 => 2024, 2024 => 2028, etc.

**Comment by WickyNilliams** [comment]
Yeah, I think this is all the same issue. I was just able to reproduce on both android and mac. I changed some logic relating to the year dropdown just before release (oops!) so I guess I introduced a bug. I will look into it now

**Comment by WickyNilliams** [comment]
PR #16

**Comment by viljamis** [comment]
Fixed in v1.0.1
